packages/Stylers/package.json

```diff
@@ -14,5 +14,8 @@
   },
   "dependencies": {
     "@paprika/tokens": "^0.1.4"
+  },
+  "peerDependencies": {
+    "styled-components": "^4.2.0"
```

Comment:
FYI technically this is a breaking change for `Stylers`.

Reply:
Do you think this might have been causing my build issue with Storybook v5.2?

Reply:
Extremely unlikely 🤔